File: Java_Programs_for_beginners/JavaNumber_Programs/NumberSystem/OddEven/OddEven.java
package NumberSystem.OddEven;

import java.util.Scanner;

public class OddEven {
    public static void main (String args[])
    {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter the Number you want to check");
        int n = in.nextInt();
        if(n%2 ==0)
        {
            System.out.println("Entered number is even");
        }
        else
        {
            System.out.println("Entered number is odd");
        }
    }
}

File: C Program/odd_even.c
#include <stdio.h>
#include<conio.h>

void main()
{
    int a;
    printf("enter a number to check even or odd\n");
    scanf("%d",&a);
    if(a%2==0){
        
        printf("%d is even",a);
    }else{
        
        printf("%d is odd",a);
        
    }
    
    getch();


}
